client/src/app/site/pages/organization/pages/accounts/components/account-main/account-main.component.ts

```diff
@@ -61,6 +64,14 @@ export class AccountMainComponent extends BaseModelRequestHandlerComponent {
         );
     }
 
+    private deleteOldModels(): void {
+        const oldModelIds = this.userController
+            .getViewModelList()
+            .map(user => user.id)
+            .filter(id => !this.accountIds.includes(id));
+        this.userRepo.deleteModels(oldModelIds);
```

Comment:
The component should not be responsible for the deletion of models in the repo. Why does the approach not work without this?

Reply:
It generally works without this, but there was one problem (which this was intended to solve):

The participant list subscribes to the participants via their parent meeting.
If I delete a participant through said list, it therefore gets reported that the participant is not in the meeting anymore, but not that he's been deleted.
If the account list (which subscribes by id) is already loaded this isn't a problem, if it isn't the user stays as a relic in the local storage and will be shown in the account list as soon as it is loaded.

This was meant to catch that by checking for the existence of any relic users before initializing the account subscription and internally deleting the models.
I agree that doing this in this place is probably wrong though, I'll try to divert this functionality to the repo and access it in a safer manner.